source/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+   api:
+       build: .
+       ports:
+       - 5000:5000
+       volumes:
+       - ${PATH_OF_INPUT}/:/app/input
+       - ${PATH_OF_OUTPUT}/:/app/output
+       - ${PATH_OF_OUTPUT}/logs:/app/output/logs
```

Comment:
Why we need to add path for ${PATH_OF_OUTPUT}/logs ?

Reply:
Ok, I will check